QUEST_LV_0100.tsv

```diff
@@ -6832,7 +6832,7 @@ QUEST_LV_0100_20150428_006831	The Highlander Master is telling you that since yo
 QUEST_LV_0100_20150428_006832	Obtain %s from Panto Archer
 QUEST_LV_0100_20150428_006833	Defeat Red Infroroctor
 QUEST_LV_0100_20150428_006834	The sealed stone told you that it won't do anything bad to you and defeat the oberserver for it. Defeat Red InfroRocter for the sealed stone.
-QUEST_LV_0100_20150428_006835	You defeated the Red InfroRocter as the stone wanted. Go back to the stone and talk to it.
+QUEST_LV_0100_20150428_006835	You defeated the Red Infro Roktor as the stone wanted. Return to the stone and talk to it.
```

Comment:
The original Infrorocktor doesn't have a space. Should we also do so with Red Infrorocktor?